.github/workflows/kvrocks.yaml

```diff
@@ -308,9 +308,17 @@ jobs:
             GOCASE_RUN_ARGS="-tlsEnable"
           fi
           ./x.py test go build $GOCASE_RUN_ARGS ${{ matrix.ignore_when_tsan}}
-
+          
+      - name: Set up Python Environment
+        run: |
+          python3 -m venv venv
+          source venv/bin/activate
+          
+          
       - name: Install redis-py
-        run: pip3 install redis==4.3.6
+        run: |
+          source venv/bin/activate
+          pip3 install redis==4.3.6
```

Comment:
Please don't change unrelated part : )

Reply:
I've updated according to the error logs in the last CI run, where it demands for a virtual environment before package installation, can you kindly check and suggest changes

Reply:
You need to change `build-and-test-in-container` instead of `build-and-test`.